src/main/java/kr/co/ksgk/ims/domain/invoice/entity/InvoiceProduct.java

```diff
@@ -21,4 +20,18 @@ public class InvoiceProduct {
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "product_id", nullable = false)
     private Product product;
+
+    private Integer returnedQuantity;
+    private Integer resaleableQuantity;
+    private String note;
+
+    private String productImageUrl;
+
+    public InvoiceProduct(Invoice invoice, Product product, Integer returnedQuantity, Integer resaleableQuantity, String note) {
+        this.invoice = invoice;
+        this.product = product;
+        this.returnedQuantity = returnedQuantity;
+        this.resaleableQuantity = resaleableQuantity;
+        this.note = note;
+    }
```

Comment:
1. productUrl 필요 없음.
2. 나머지 필드는 규칙에 Column 어노테이션 추가
3. resaleable -> resalable 고침

```suggestion
    @Column(nullable = false)
    private Integer returnedQuantity;

    @Column(nullable = false)
    private Integer resalableQuantity;

    @Lob
    private String note;

    @Builder
    public InvoiceProduct(Invoice invoice, Product product, Integer returnedQuantity, Integer resalableQuantity, String note) {
        this.invoice = invoice;
        this.product = product;
        this.returnedQuantity = returnedQuantity;
        this.resalableQuantity = resalableQuantity;
        this.note = note;
    }
```